### ISSUE TITLE
%a is not taken into account in org-roam-dailies-capture-templates

### ISSUE THREAD

**Issue by brabalan** [org-roam/org-roam#1277]
### Description
Setting up an org-roam capture template with `%a` does not put a link to the current item, as `org-capture` does.

#### Steps to Reproduce
With the following configuration:
```emacs-lisp
  (setq org-roam-dailies-capture-templates
        '(("j" "journal" entry
           #'org-roam-capture--get-point
           "* %<%H:%M> %?"
           :file-name "daily/%<%Y-%m-%d>"
           :head "#+title: %<%Y-%m-%d>\n\n* [/] Do Today"
           :olp ("Journal"))
          ("t" "do today" item
           #'org-roam-capture--get-point
           "[ ] %a"
           :file-name "daily/%<%Y-%m-%d>"
           :head "#+title: %<%Y-%m-%d>\n\n* [/] Do Today"
           :immediate-finish t
           :olp ("Do Today")))))
```
run `org-roam-dailies-capture-today` on a task you want to plan for the day and choose tempate `t`.

#### Expected Results
My daily file has
```org
* [0/1] Do Today
- [ ] [[file:~/Documents/Org/todo-orgx.org::*Watch \[\[https://emacsconf.org/2020/schedule/\]\[EmacsConf - 2020 - Schedule\]\]][Watch EmacsConf - 2020 - Schedule]]
```

#### Actual Results
The `%a` is not expanded and I get
```org
* [0/1] Do Today
- [ ] 
```

### Environment
 - Emacs: GNU Emacs 27.1 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.24.22, cairo version 1.17.3)
 of 2020-08-28
- Framework: Doom
- Org: Org mode version 9.5 (nil @ /home/schmitta/.emacs.d/.local/straight/build-27.1/org-mode/)
- Org-roam: 1.2.3
- Org-roam commit: https://github.com/jethrokuan/org-roam/commit/8401784

**Comment by jethrokuan** [comment]
This should now be fixed with #1347

**Comment by brabalan** [comment]
I confirm it works. Thanks!
